src/main/java/hudson/util/VersionNumber.java

```diff
@@ -514,16 +514,17 @@ public boolean isNewerThan(VersionNumber rhs) {
     }
 
     /**
-     * @deprecated see {@link #getDigitAt(int)}
+     * Returns the nth integer component of the version string. The first element is index 0. Non-integer items are skipped. Returns the last integer component if the index is too big.
      */
     public int digit(int idx) {
         Iterator i = items.iterator();
```

Comment:
What it happens with `digit(0)`? Part of the motivation behind of https://github.com/jenkinsci/lib-version-number/pull/3 was to provide a more robustness method by providing a better input validation.